package/utils/rbextract/src/rbextract.c

```diff
@@ -448,7 +451,9 @@ main(int argc, char **argv)
 			printf("Board product code: %s\n", rb_get_board_product_code());
 			printf("Board identifier: %s\n", rb_get_board_identifier());
 			printf("Board serial: %s\n", rb_get_board_serial());
-			printf("Board MAC: %X\n", rb_get_board_mac());
+			mac = rb_get_board_mac();
+			printf("Board MAC: %02x:%02x:%02x:%02x:%02x:%02x\n",
+					mac.bytes[0], mac.bytes[1], mac.bytes[2], mac.bytes[3], mac.bytes[4], mac.bytes[5]);
```

Comment:
The correct size specifier should be 02hhx ?

Reply:
True, they're getting promoted. Replaced with portable PRIx8 macros.